electron/routing_service.ts

```diff
@@ -106,31 +87,26 @@ export class WindowsRoutingService implements RoutingService {
         const netErr = err as NetError;
         if (netErr.errno === 'ENOENT') {
           console.info(`Routing service not running. Attempting to start.`);
-          // Prompt the user for admimn permissions to start the routing service.
+          // Prompt the user for admin permissions to start the routing service.
           sudo.exec(SERVICE_START_COMMAND, {name: 'Outline'}, (sudoError, stdout, stderr) => {
             if (sudoError) {
-              if (/service has already been started|net helpmsg 2182/i.test(sudoError.message)) {
-                // Wait for the service to start before sending the request.
```

Comment:
IIRC, the timeout was necessary to give the service a chance to start. Otherwise, there could be a race between the service starting up and sending the request. However, if you have seen it succeed without it in your testing, let's remove it.

Reply:
No, I haven't seen this and I just tested on an ancient Windows 8 laptop to confirm. `net start` seems to wait before exiting.